common/step_download.go

```diff
@@ -208,20 +192,21 @@ func (s *StepDownload) download(ctx context.Context, ui packer.Ui, source string
 	}
 
 	ui.Say(fmt.Sprintf("Trying %s", u.String()))
-	gc := getter.Client{
-		Ctx:              ctx,
+	req := &getter.Request{
 		Dst:              targetPath,
 		Src:              src,
 		ProgressListener: ui,
 		Pwd:              wd,
-		Dir:              false,
-		Getters:          getters,
+		Mode:             getter.ModeFile,
+	}
+	if runtime.GOOS == "windows" {
+		req.Inplace = true
```

Comment:
Eh I wonder why I put this in an if; this would be handy on other systems too. To not create a symlink.

Reply:
I'll remove it